src/app/public/modules/modal/modal-header.component.scss

```diff
@@ -0,0 +1,11 @@
+@import "~@skyux/theme/scss/mixins";
+@import "~@skyux/theme/scss/_compat/mixins";
+
+// Modal header should mimic a regular paragraph with .sky-emphasized applied.
```

Comment:
Simply adding the `.sky-emphasized` CSS class to the `h1` wouldn't work?

Reply:
I was worried about coupling them since it seems `.sky-emphasized` is intended for paragraph copy. On second look, the class does already have overrides for the font-weight, size, and family, so I think you're probably on to a good idea here.
I've updated it.